src/site/index.scss

```diff
@@ -1,5 +1,11 @@
 @use '@moduk/frontend/src/css' with (
-  $govuk-new-link-styles: true
+  $govuk-new-link-styles: true,
+  // light maroon
+  $govuk-link-colour: #643f58,
+  // dark maroon
+  $govuk-link-hover-colour: #4b263e,
+  // maroon
+  $govuk-link-active-colour: #532a45
```

Comment:
We have and chicken and egg scenario here, I wanted to override the `gov-links` colours but I don't have access to`gov-colour('maroon')` at this point. Any suggestion?

Reply:
Don't think we can do it that way as that will affect the components. I'm using a `moduk-link--light-maroon` modifier class in #29.